core/src/trezor/ui/scroll.py

```diff
@@ -32,6 +32,11 @@ def render_scrollbar(pages: int, page: int) -> None:
 
 
 def render_swipe_icon() -> None:
+    if utils.DISABLE_ANIMATION:
```

Comment:
istm it would be nicer to limit the `DISABLE_ANIMATION` code just to set the value of `c` to a constant

Reply:
Fixed in https://github.com/trezor/trezor-firmware/pull/784/commits/47419c77688a66c5d32b06edbc8f0afbdb33b95f